src/test/java/org/jboss/arquillian/extension/jacoco/test/unit/ApplicationArchiveInstrumenterTest.java

```diff
@@ -0,0 +1,44 @@
+package org.jboss.arquillian.extension.jacoco.test.unit;
+
+
+import org.jboss.arquillian.core.api.Instance;
+import org.jboss.arquillian.extension.jacoco.client.ApplicationArchiveInstrumenter;
+import org.jboss.arquillian.extension.jacoco.client.JacocoConfiguration;
+import org.jboss.arquillian.test.spi.TestClass;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.spec.EnterpriseArchive;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+
+import java.util.HashMap;
+
+import static org.mockito.Mockito.*;
+
+@RunWith(org.mockito.junit.MockitoJUnitRunner.class)
+public class ApplicationArchiveInstrumenterTest {
+
+    @Mock
+    private Instance<JacocoConfiguration> config;
+
+    @Test
+    public void should_not_break_process_archive_if_it_contains_path_as_ear_type(){
+
+        when(config.get()).thenReturn(JacocoConfiguration.fromMap(new HashMap<String, String>()));
```

Comment:
I have a feeling that employing mockito is a bit too much for this case. We can simply pass `JacocoConfiguration.fromMap(new HashMap<String, String>())` directly through the setter (if we stick to it) - see for example `AxuiliaryArchiveAppenderTestCase` (btw can you fix typo in this class name? should be `Aux...`)